src/Sarif.Driver/Sdk/SarifLogger.cs

```diff
@@ -64,20 +63,35 @@ public class SarifLogger : IDisposable, IAnalysisLogger
                 }
             }
 
-            string invocation = Environment.CommandLine;
+
+            run.Invocation = Invocation.Create();
 
             if (invocationTokensToRedact != null)
             {
-                foreach (string tokenToRedact in invocationTokensToRedact)
+                run.Invocation.Machine = Redact(run.Invocation.Machine, invocationTokensToRedact);
```

Comment:
I don't understand. If we consider machine name to be sensitive information, then why fill it in at all?